bridge/bridge.py

```diff
@@ -19,6 +19,11 @@ def __init__(self):
         self.debug = False
         self.name = 1  # Change when using a new bridge!
         self.cloud = 'http://127.0.0.1:5000'
+        """generate token using:
+        token = jwt.encode(
+            {'public_id': self.name, 'exp' : datetime.datetime.utcnow() + datetime.timedelta(days=8)}, 'my-secret')
+        use secret from bridge"""
+        self.accessToken = ''
```

Comment:
Maybe the comment above should also say that the token should not be just filled in this variable. This empty strink looks a little bit like an invitation :sweat_smile:

Reply:
According to [this](https://cloud.google.com/docs/authentication/api-keys), the main goal is to keep the secret outside of the project repo. Therefore, an environment variable should be fine :+1:

Reply:
thank you!